Basic-Car-Maintenance/Shared/Models/AlertItem.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AlertItem.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Omar Hegazy on 13/10/2023.
+//
+
+import Foundation
+import FirebaseFirestoreSwift
+import SwiftData
+
+struct AlertItem: Codable, Identifiable {
+    @DocumentID var id: String?
+    var actionTxt: String
```

Comment:
Also, cause it's show short, just spell out `actionText` rather than `actionTxt`